index.bs

```diff
@@ -263,6 +263,30 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+<h3 id="backwards-compat">Support Existing Content</h3>
+
+Existing content often relies upon expected user agent processing and behavior to function as intended.
+Processing requirements should be specified to ensure that user agents will be able to handle most existing content.
+In particular, it should be possible to process existing web content in current user agents and get results that are compatible with the existing expectations of users and authors,
+based on the behavior of existing browsers.
+
+Content relying on existing browser behavior can take many forms.
+It may rely on elements, attributes or APIs that are part of earlier specifications, or on features that are entirely proprietary.
+It may depend on specific error handling rules. It may depend on certain method names not being defined.
```

Comment:
Do we have any example of that?

Reply:
Lots of example is UA sniffing content. Not saying this is a good example but the tail end of the web still depends on it.